internal/services/api/post_inspect_remote_test.go

```diff
@@ -0,0 +1,76 @@
+package api
+
+// Copyright (C) 2023 by Posit Software, PBC.
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/posit-dev/publisher/internal/accounts"
+	"github.com/posit-dev/publisher/internal/config"
+	"github.com/posit-dev/publisher/internal/logging"
+	"github.com/posit-dev/publisher/internal/types"
+	"github.com/posit-dev/publisher/internal/util"
+	"github.com/posit-dev/publisher/internal/util/utiltest"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/suite"
+)
+
+type PostInspectRemoteSuite struct {
+	utiltest.Suite
+	cwd util.AbsolutePath
+}
+
+func TestPostInspectRemoteSuite(t *testing.T) {
+	suite.Run(t, new(PostInspectRemoteSuite))
+}
+
+func (s *PostInspectRemoteSuite) SetupTest() {
+	fs := afero.NewMemMapFs()
+	cwd, err := util.Getwd(fs)
+	s.Nil(err)
+	s.cwd = cwd
+	s.cwd.MkdirAll(0700)
+}
+
+func (s *PostInspectRemoteSuite) TestPostInspectRemote() {
```

Comment:
It might be good to also have tests for some of the non-happy paths. E.g: Unexpected error from `accountList.GetAccountByName` and when it is the known error `ErrAccountNotFound`